docs/code-convention.md

```
@@ -0,0 +1,110 @@
+  e.g. `chore/update-eslint-config-77`
+- **test/** → 테스트 코드 추가/수정  
+  e.g. `test/user-service-coverage-331`
+- **release/** → 배포 준비 브랜치  
+  e.g. `release/v1.0.0`
+
+---
+
+## 📩 Commit Convention
+
+### 1. Commit Message Rules
+- 기본 규칙: **[기능 Type][GitHub 닉네임]: 작업 내용 (#이슈번호 선택)**
```

markdownlint-cli2 (0.17.2)

Reference links and images should use a label that is defined
Missing link or image reference definition: "github 닉네임"

(MD052, reference-links-images)